organizations/migrations/0004_auto_20230727_1206.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 3.2.20 on 2023-07-27 12:06
```

Comment:
This migration is invalid. 0004 already exists in migrations folder. You could rename it to 0008...